2025-04-10 17:35:27,236 - ERROR - Unexpected error: BaseEventLoop.create_connection() got an unexpected keyword argument 'extra_headers'
2025-04-10 17:35:27,238 - ERROR - Max reconnection attempts reached
2025-04-10 17:40:43,120 - ERROR - Unexpected error: BaseEventLoop.create_connection() got an unexpected keyword argument 'extra_headers'
2025-04-10 17:40:43,120 - ERROR - Max reconnection attempts reached
